doc/source_debugging.md

```diff
@@ -76,21 +98,21 @@ make -j $(nproc)
 ``` bash
 cd ${WAMR_ROOT}/product-mini/platforms/linux
 mkdir build && cd build
-cmake .. -DWAMR_BUILD_DEBUG_AOT=1
+cmake .. -DCMAKE_BUILD_TYPE:STRING="Debug" -DWAMR_BUILD_DEBUG_AOT=1
 make
 ```
 
 4. Compile wasm module to AOT module
 ``` bash
-wamrc -o test.aot test.wasm
+${WAMR_ROOT}/wamr-compiler/build/wamrc --opt-level=0 --size-level=0 -o test.aot test.wasm
```

Comment:
Must we add `--opt-level=0 --size-level=0`?

Reply:
Yes, @maht could you help confirm it? Since `--opt-level=0 --size-level=0` flags for wamrc impact the AOT performance a lot, we had better remove them if they are not necessary.